lib/vagrant-libvirt/action/create_network_interfaces.rb

```diff
@@ -245,6 +247,7 @@ def interface_xml(type, source_options, mac, device_name,
               xml.target(dev: target_dev_name(device_name, type, iface_number))
               xml.alias(name: "net#{iface_number}")
               xml.model(type: model_type.to_s)
+              xml.mtu(size: mtu) if mtu
```

Comment:
Suggest using `mtu.to_i` to make sure this is an integer at this point, or validate that it is either `nil` or an integer in the configuration in the config object.

Reply:
I decided to use the Integer Constructor, since to_i would cast any string to 0 if its not a valid integer. I prefer an exception, if the input is not a valid integer instead. If that is not what you had in mind, I would write a validator function instead (e.g. new module util/validators & function validate_integer) to raise a custom validation error.